protocol/x/clob/memclob/memclob.go

```diff
@@ -1216,6 +1216,18 @@ func (m *MemClobPriceTimePriority) PurgeInvalidMemclobState(
 		m.RemoveOrderIfFilled(ctx, orderId)
 	}
 
+	// For order matches in the last block, send an orderbook update
+	// to the grpc streams.
+	// Note that fully filled orders are removed from the orderbook in `RemoveOrderIfFilled` above.
+	allUpdates := types.NewOffchainUpdates()
+	for _, orderId := range filledOrderIds {
+		if m.openOrders.hasOrder(ctx, orderId) {
+			orderbookUpdate := m.GetOrderbookUpdatesForOrderUpdate(ctx, orderId)
+			allUpdates.Append(orderbookUpdate)
+		}
+	}
+	m.clobKeeper.SendOrderbookUpdates(allUpdates, false)
```

Comment:
Would it make sense to move this from PurgeInvalidMemclobState to PCS in  abci.go? It seems that we depend on `filledOrderIds`, which is read off of `		processProposerMatchesEvents.OrderIdsFilledInLastBlock`. PCS is the main driver for using PPME and i don't think sending updates is related to purging memclob state?